modules/gitops-argocd-cli-creating-an-application-in-normal-mode.adoc

```diff
@@ -0,0 +1,133 @@
+// Module included in the following assemblies:
+//
+// * argocd_applications/creating-an-application-using-gitops-argocd-cli.adoc
+// * declarative_clusterconfig/configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations.adoc
+
+ifeval::["{context}" == "configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations"]
+:cluster:
+endif::[]
+ifeval::["{context}" == "creating-an-application-using-gitops-argocd-cli"]
+:app:
+endif::[]
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-argocd-cli-creating-an-application-in-normal-mode_{context}"]
+= Creating an application in normal mode by using the GitOps CLI
+
+You can create applications in normal mode by using the {gitops-shortname} `argocd` CLI.
+
+ifdef::cluster[]
+This sample workflow walks you through the process of configuring Argo CD to recursively sync the content of the `cluster` directory to the `cluster-configs` application. The directory defines the {OCP} cluster configurations and a namespace `spring-petclinic` on the cluster.
+endif::cluster[]
+
+.Prerequisites
+
+* You have installed the {gitops-title} Operator in your cluster.
+* You have logged into Argo CD instance.
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the {gitops-title} `argocd` CLI. 
+
+.Procedure
+
+. Get the admin account password for the Argo CD server:
++
+[source,terminal]
+----
+$ ADMIN_PASSWD=$(oc get secret openshift-gitops-cluster -n openshift-gitops -o jsonpath='{.data.admin\.password}' | base64 -d)
+----
+
+. Get the Argo CD server URL:
++
+[source,terminal]
+----
+$ SERVER_URL=$(oc get routes openshift-gitops-server -n openshift-gitops -o jsonpath='{.status.ingress[0].host}')
+----
+
+. Log in to the Argo CD server:
++
+.. Display the line of text or string of the admin account password:
++
+[source,terminal]
+----
+$ echo ${ADMIN_PASSWD}
```

Comment:
nit: Is displaying the password required? We log in using the variable in the next step

Reply:
@anandf Can you please comment on this and provide the clarification?

Reply:
Yes, I will remove it. Probably its providing too much information. Customers can find a way to get the password and paste the value in single quotes.